apps/docs/src/app/core/component-docs/wizard/examples/wizard-branching-example.component.html

```diff
@@ -10,12 +10,14 @@
             <!-- The Wizard -->
             <fd-wizard>
                 <fd-wizard-navigation>
-                    <ul fd-wizard-progress-bar size="xl">
+                    <ul fd-wizard-progress-bar size="xl" role="list" aria-label="Wizard Steps">
```

Comment:
same question as for navigation. Why do you need the role as it's a native `ul` element?